pkg/zeaburpack/main.go

```diff
@@ -195,6 +196,9 @@ func Build(opt *BuildOptions) error {
 		_ = os.RemoveAll(*opt.Path + "/.zeabur")
 	}
 
+	// try to copy .zeabur directory from the image to the host, ignore error because it's fine if it not exists
+	_ = utils.CopyFromImage(*opt.ResultImage, "/src/.zeabur/.", path.Join(*opt.Path, ".zeabur"))
```

Comment:
Not sure why we have a dot after `.zeabur/`.

Reply:
`.zeabur` means the `.zeabur` directory, `zeabur/.` means all files inside `.zeabur` directory.